### PR TITLE
Prepare for plone.app.dexterity docs

### PR DESCRIPTION
**Pull request by stevepiercy**
Refs: https://github.com/plone/plone.app.dexterity/issues/360

### CODE REVIEW

File: docs/contributing/admins.md [hunk 1]
```diff
@@ -15,6 +15,42 @@ This guide is for administrators of Plone Documentation.
...
+
+    ```shell
+    bin/pip install "rst-to-myst[sphinx]"
+    ```
+
+1.  Clone the project repository to the root of `plone/documentation`.
+1.  Delete any non-documentation files from the clone.
+1.  Move the documentation files and subfolders to the root of the clone, retaining the documentation structure.
+1.  Convert the reStructuredText documentation files to MyST.
+    The example commands below assume that there are files at the root of the clone and in one sub-level of nested directories. For deeper nesting, insert globbing syntax for each sub-level as `**/`
```
Comment (jensens) [inline docs/contributing/admins.md]:
```suggestion
    The example commands below assume that there are files at the root of the clone and in one sub-level of nested directories.
    For deeper nesting, insert globbing syntax for each sub-level as `**/`
```

Comment (stevepiercy) [reply docs/contributing/admins.md]: Good catch. Anything else? OK to merge?

Comment (stevepiercy) [reply docs/contributing/admins.md]: Whoops. nm. Too early. I still have a lot more work ahead with cleaning up the converted MyST files.

**Comment by stevepiercy** [comment]
This is now ready to merge. I'm going ahead to merge it, as it does not change deployed content, but it establishes `plone.app.dexterity` in MyST, providing content experts with a base to start migrating content into the Plone 6 documentation. Any content edits can be reviewed as they come in through new PRs.

**Pull request merged**